openshift/templates/patroni.yaml

```diff
@@ -353,31 +305,3 @@ parameters:
     displayName: Replication Username
     name: PATRONI_REPLICATION_USERNAME
     value: replication
-  - description: Service account name used for pods and rolebindings to form a cluster in the project.
-    displayName: Service Account
-    name: SERVICE_ACCOUNT
-    value: patroniocp
-  - description: The size of the persistent volume to create.
-    displayName: Persistent Volume Size
-    name: PVC_SIZE
-    value: 5Gi
-  - description: Version of Patroni image to be used.
-    displayName: Patroni image version
-    name: IMAGE_VERSION
-    value: "v10"
-  - description: Pod management policy. For faster startups, use Parallel.
-    displayName: Pod management policy.
-    name: POD_MANAGEMENT_POLICY
-    value: "Parallel"
-  - description: Rocket.Chat channel to use for notifications.
```

Comment:
Have you configured notifications in Silver to warn us when we're approaching our disk space limit? I seem to recall reading a while ago that this was one of the things that OCP4 allows that OCP3 doesn't

Reply:
@andrea-williams I've decided to just forego it entirely right now, in preference to us just using the bcgov image, no questions asked. (They just fail the health check at 90% - which would be 22.5GB for us right now).
I remember some talk of Prometheus being available with OCP4 - I don't know where that stands.